forth_src/io.fs

```diff
@@ -0,0 +1,79 @@
+require open
+
+\ Set input device to open file #
+\ returns 0 on success, file # on error
```

Comment:
This comment looks wrong, should it be "\result=0 on success, or CHKIN error code" ?